api_tests/dry/rfc003/test.js

```diff
@@ -374,4 +375,77 @@ describe("RFC003 HTTP API", () => {
             "Rejected"
         );
     });
+
+    let alice_swap_with_charlie_href;
+    it("[Alice] Should be able to send a swap request to a different node via HTTP api", async () => {
+        await chai
+            .request(alice.comit_node_url())
+            .post("/swaps/rfc003")
+            .send({
+                alpha_ledger: {
+                    name: alpha_ledger_name,
+                    network: alpha_ledger_network,
+                },
+                beta_ledger: {
+                    name: beta_ledger_name,
+                },
+                alpha_asset: {
+                    name: alpha_asset_name,
+                    quantity: alpha_asset_reasonable_quantity,
+                },
+                beta_asset: {
+                    name: beta_asset_name,
+                    quantity: beta_asset_quantity,
+                },
+                alpha_ledger_refund_identity: null,
+                beta_ledger_redeem_identity: alice_final_address,
+                alpha_ledger_lock_duration: alpha_ledger_lock_duration,
+                peer_address: charlie_comit_node_address
+            })
+            .then(res => {
+                res.error.should.equal(false);
+                res.should.have.status(201);
+                swap_location = res.headers.location;
+                swap_location.should.be.a("string");
+                alice_swap_with_charlie_href = swap_location;
+            });
+    });
+
+    it("[Alice] Should be in Start state after sending the swap request to Charlie", async function() {
+        await alice.poll_comit_node_until(
+            chai,
+            alice_swap_with_charlie_href,
+            "Start"
+        );
+    });
+
+    it("[Charlie] Shows the Swap as Start in /swaps", async () => {
+        let res = await chai.request(charlie.comit_node_url()).get("/swaps");
+
+        let embedded = res.body._embedded;
+        let swap_embedded = embedded.swaps[0];
+        swap_embedded.protocol.should.equal("rfc003");
+        swap_embedded.state.should.equal("Start");
+        let swap_link = swap_embedded._links;
+        swap_link.should.be.a("object");
+        let charlie_swap_href = swap_link.self.href;
+        charlie_swap_href.should.be.a("string");
+    });
+
+    it("[Alice] Should see both Bob and Charlie in her list of peers after sending a swap request to both of them", async () => {
+        await chai
+            .request(alice.comit_node_url())
+            .get("/peers")
+            .then(res => {
+                res.should.have.status(200);
+
+                let peers = res.body.peers;
+                for (peer of peers) {
```

Comment:
❌ I see two issues here: 
1) if `peers` is empty, this test passes
2) if `peers` contains `charlie_comit_node_address` or `bob_comit_node_address` twice, this test passes.

This should be fixed please.